.gitlab/lassen-build-and-test-extra.yml

```diff
@@ -67,10 +67,23 @@ xl_16_1_1_12_cuda_11_1_1:
   allow_failure: true
   extends: .build_and_test_on_lassen
 
+# Overriding shared spec: Deactivated spec. This spec will be removed soon.
```

Comment:
Should we remove this spec now, rather than letting it fail, and add one for a newer CUDA version?

Reply:
That would require a change to the radiuss-shared-ci project but I'm for it

Reply:
@rhornung67 With the code below, this spec is not running anything: the script section of the shared job has been overridden. No resource wasted.
I encourage you to submit an issue to Radiuss-Shared-CI. We have have it changed fairly quickly. I just want this to be a clean reviewed process.

Reply:
E.g. I want @davidbeckingsale’s approval on the radiuss-shared-ci issue saying that we don’t need it for Umpire.